Bottom: a58522a6982c344af95669e7cb0b4f70dac6fcb6
Top:    39acb64757a9357412f01b544453d1c5da41833e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 16:49:05 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 3756955..33b7650 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1190,6 +1190,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1212,6 +1213,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
